src/interface/variable.hpp

```diff
@@ -118,7 +118,9 @@ class CellVariable {
   inline std::string getAssociated() { return m_.getAssociated(); }
 
   KOKKOS_FORCEINLINE_FUNCTION
-  std::size_t GetUniqueID() const { return uid_; }
+  Uid_t GetUniqueID() const { return uid_; }
+
+  static Uid_t GetUniqueID(const std::string &var_label) { return get_uid_(var_label); }
```

Comment:
It's a little unintuitive that this *sets* the unique ID if it's not present (my fault for the naming) but I think that's probably fine. It's a greedy get.

Reply:
Is there a conventional name for a greedy get? It seems like a word that should exist but I couldn't think of one.

Reply:
Yeah same.